evergreen/config.yml

```diff
@@ -1120,6 +1120,56 @@ tasks:
       script: |-
         ${cmake_build_type|Debug}/realm-libfuzz -rss_limit_mb=0 -max_total_time=3600
 
+- name: generate-sync-corpus
+  tags: [ "for_nightly_tests" ]
+  exec_timeout_secs: 1800
+  commands:
+  - func: "fetch source"
+  - func: "fetch binaries"
+  - func: "compile"
+    vars:
+      target_to_build: "SyncTests"
+  - command: shell.exec
+    params:
+      working_dir: realm-core
+      shell: bash
+      script: |-
+          set -o errexit
+          set -o verbose
+
+          if [[ -n "${c_compiler}" && "$(basename ${c_compiler})" = "clang" && -f "$(dirname ${c_compiler})/llvm-symbolizer" ]]; then
+              LLVM_SYMBOLIZER="$(dirname ${c_compiler})/llvm-symbolizer"
+              export ASAN_SYMBOLIZER_PATH="$(./evergreen/abspath.sh $LLVM_SYMBOLIZER)"
+              export TSAN_OPTIONS="external_symbolizer_path=$(./evergreen/abspath.sh $LLVM_SYMBOLIZER)"
+          fi
+
+          export UNITTEST_EVERGREEN_TEST_RESULTS="$(./evergreen/abspath.sh ${task_name}_results.json)"
+          if [[ -n "$UNITTEST_EVERGREEN_TEST_RESULTS" && -f "$UNITTEST_EVERGREEN_TEST_RESULTS" ]]; then
+              rm "$UNITTEST_EVERGREEN_TEST_RESULTS"
+          fi
+          export UNITTEST_PROGRESS=1
+          export UNITTEST_FILTER="Array_Example Transform_* EmbeddedObjects_*"
+          export UNITTEST_DUMP_TRANSFORM="changeset_dump"
+
+          export TSAN_OPTIONS="suppressions=$(pwd)/test/tsan.suppress history_size=4 $TSAN_OPTIONS"
+          export UBSAN_OPTIONS="print_stacktrace=1"
+
+          ./build/test/${cmake_build_type}/realm-sync-tests
```

Comment:
Local relative dir is `realm-core`?

Reply:
Yup. You can see the working directory for the task under the `working_dir` param for each `shell.exec` block in the evergreen config.